Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.6-slim-stretch
+
+# install libsundials-serial-dev
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y libsundials-serial-dev
+
+# install nginx
```

Comment:
Sorry I don't really know much about this. What is this doing?

Reply:
this is just installing nginx, which is used to serve the static files (gunicorn is used to serve the dynamic pages). The combination of nginx for static files and gunicorn for dynamic content seems quite popular, so I was just following this.

Reply:
Ok cool, thanks